OpenRA.Mods.RA2/Traits/ChangeOwnerOnGarrisoner.cs

```diff
@@ -44,16 +50,17 @@ void INotifyGarrisonerEntered.OnGarrisonerEntered(Actor self, Actor garrisoner)
 				return;
 
 			NeedChangeOwner(self, garrisoner, newOwner);
-
-			Game.Sound.PlayNotification(self.World.Map.Rules, garrisoner.Owner, "Sounds", info.EnterNotification, newOwner.Faction.InternalName);
+			Game.Sound.Play(SoundType.World,info.EnterSound,self.CenterPosition);
```

Comment:
Missing spaces.

Run `make check` locally, please.

Reply:
Make check has been run, style cop now reports no issue :) should be good to go.